File: cpuheavy/cpuheavy.js
const express = require("express");
const app = express();
const port = 1236;

function fibonacci(num) {
  if (num == 0) {
    return 0;
  }
  if (num == 1) {
    return 1;
  }
  return fibonacci(num - 1) + fibonacci(num - 2);
}

app.get("/", (req, res) => {
  var tmp = fibonacci(45);
  res.send(tmp.toString());
});

app.listen(port, () => console.log(`Example app listening on port ${port}!`));

File: cpuheavy/cpuheavy.py
from flask import Flask
import time
app = Flask(__name__)


def fibonacci(num):
    if num <= 1:
        return num
    return fibonacci(num-1) + fibonacci(num-2)


@app.route("/")
def hello():
    tmp = fibonacci(45)
    return str(tmp)


app.run(threaded=True)
